tests/Gaufrette/Functional/LocalDirectoryDeletor.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Gaufrette\Functional;
+
+class LocalDirectoryDeletor
+{
+    public static function deleteDirectory($directory)
+    {
```

Comment:
Would it make sense to check that `$directory` is not equal to `/`, as a basic fail-safe?